marklogic-unit-test-client/src/test/ml-modules/root/test/suites/JavaScript Modules/assertTrue.mjs

```diff
@@ -0,0 +1,47 @@
+const test = require("/test/test-helper.xqy");
+
+const list = fn.head(xdmp.apply(xdmp.function(fn.QName("http://marklogic.com/test", "list"), "/test/test-controller.xqy")));
+
+// TODO - Should "assertThrowsError" & "assert-equal-json.assertThrowsErrorWithMessage" be made public? I don't see a public JS version of those assertions.
```

Comment:
Looks like something should exist - copy/pasting it here is fine, just remove this comment as we talked about this offline.